corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -393,6 +393,11 @@ class StackDatum(IdNode):
 
     value = XPathField('@value')
 
+class StackInstanceDatum(IdNode):
```

Comment:
nit: you should fix lint errors that are introduced by the PR

Reply:
this one is [fixed](https://github.com/dimagi/commcare-hq/pull/33275/commits/6b52c211159055cae4422b5e9e9521b53e97c586), although I am ignoring the line too long errors in test files with xml. Let me know though if I should not.